configure.ac

```diff
@@ -793,6 +793,9 @@ if test "$cross_compiling" = yes; then
 	*-*-vxworks*)
 		_host_ident=$host_cpu
 		;;
+  *-*-emscripten)
+    _host_ident=$(emcc -dumpversion)-$host_cpu
+    ;;
```

Comment:
There are some indentation inconsistency issues here. I know the existing configure.ac is a bit of a mess when it comes to spaces vs tabs, but at least we can avoid making the problem worse.

Reply:
Fixed, thanks.